src/routes/task.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express')
+const router = express.Router()
+const path = require('path')
+const pg = require('pg')
+const task = require('../database/queries/task.js')
+const user = require('../database/queries/users.js')
+
+
+router.get('/', (request, response, next) => {
+  const currentUser = request.user
+  user.findByHandle(currentUser.handle)
+  .then( returnedUser => {
+    task.getBy('user_id', returnedUser.id)
+    .then( response.json )
+  })
+  .catch( err )
```

Comment:
this `catch` is not really doing anything, since it is returning the error. You probably want to `console.err` the error with a custom message, so that you could look at the logs to figure out which function caused an exception to occur. 

The custom messaged logged could be something like `Error getting user with github_handle:${currentUser.handle} from the db`